satpy/tests/reader_tests/test_seviri_l1b_nc.py

```diff
@@ -285,6 +332,9 @@ def test_get_dataset(self, file_handler, channel, calibration):
         expected['acq_time'] = ('y', [np.datetime64('1958-01-02 00:00:01'),
                                       np.datetime64('1958-01-02 00:00:02')])
         expected = expected[::-1]  # reader flips data upside down
+        if mask_bad_quality_scan_lines:
+            expected = file_handler._mask_bad_quality(expected, dataset_info)
```

Comment:
I know I'm being a bit picky, but you're using the method under test to compute the expected values ;) Can you define them explicitly?

Reply:
I probably could. In a discussion with @mraspaud at PCW we found that it is ok to use it here because `_mask_bad_quality` is tested in a separate unit test on its own. Thus if the test passes it can safely be used here.

Reply:
Ok, sounds good!